examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-ingress
+---  
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-ingress-serviceaccount
```

Comment:
@nevetS l think you can review the ingress's source code, and then you can write a streamline rabc yaml.